### PR TITLE
Security fixes

### PR DESCRIPTION
**Pull request by Anusha913082**

### CODE REVIEW

File: admin/class-wpadcenter-admin.php [hunk 1]
```diff
@@ -1373,7 +1373,7 @@ public function wpadcenter_manage_ads_column_values( $column = '', $ad_id = 1 )
 			case 'stats-for-today':
 				$today = gmdate( 'Y-m-d' );
 				global $wpdb;
-				$results = $wpdb->get_results( $wpdb->prepare( 'SELECT * FROM ' . $wpdb->prefix . 'ads_statistics where ad_date=%s AND ad_id=%d', array( $today, $ad_id ) ) ); // phpcs:ignore
+				$results = $wpdb->get_results( $wpdb->prepare( 'SELECT * FROM ' . $wpdb->prefix . 'ads_statistics where ad_date=%s AND ad_id=%d', array( $today, $ad_id ) ) ); //phpcs:ignore
```
Comment (dhananjaymayane) [inline admin/class-wpadcenter-admin.php]: please add appropriate key for the phpcs ignore

File: admin/class-wpadcenter-admin.php [hunk 2]
```diff
@@ -2502,7 +2503,7 @@ public function wpadcenter_ad_selected() {
 		if ( 'selected_ad_reports' === $_POST['action'] ) {
 			$start_date = isset( $_POST['start_date'] ) ? gmdate( 'Y-m-d', intval( $_POST['start_date'] ) ) : '';
 			$end_date   = isset( $_POST['end_date'] ) ? gmdate( 'Y-m-d', intval( $_POST['end_date'] ) ) : '';
-			$ads = $_POST['selected_ad']; // phpcs:ignore
+			$ads        = isset( $_POST['selected_ad'] ) ? wp_unslash( $_POST['selected_ad'] ) : '';// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```
Comment (dhananjaymayane) [inline admin/class-wpadcenter-admin.php:2506]: add iterative sanitization on the array

File: admin/class-wpadcenter-admin.php [hunk 3]
```diff
@@ -3336,7 +3340,7 @@ public function wpadcenter_adgroup_gutenberg_preview() {
 		}
 		$adgroup_ids = array();
 		if ( ! empty( $_POST['ad_groups'] ) ) {
-			$adgroup_ids = $_POST['ad_groups']; //phpcs:ignore
+			$adgroup_ids = wp_unslash( $_POST['ad_groups'] );// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```
Comment (dhananjaymayane) [inline admin/class-wpadcenter-admin.php:3358]: Add iterative sanitization on array.

File: admin/class-wpadcenter-admin.php [hunk 4]
```diff
@@ -3438,7 +3452,7 @@ public function wpadcenter_random_ad_gutenberg_preview() {
...
```
Comment (dhananjaymayane) [inline admin/class-wpadcenter-admin.php:3472]: Add iterative sanitization for the array

**Review by dhananjaymayane — COMMENTED** [review]
Please resolve the comments, and add proper commit message for the PR's

**Review by dhananjaymayane — APPROVED** [review]

**Review by pranayhbwsl — APPROVED** [review]

**Review by dhananjaymayane — APPROVED** [review]

**Pull request merged**